test/src/unit-class_const_iterator.cpp

```diff
@@ -220,20 +220,20 @@ TEST_CASE("const_iterator class")
             {
                 json j(json::value_t::null);
                 json::const_iterator it = j.cbegin();
-                CHECK(it.m_it.primitive_iterator == 1);
+                CHECK(it.m_it.primitive_iterator.m_it == 1);
```

Comment:
Thanks for the explanation. Sorry for not having seen this in the discussion above. Great work!